packages/api/test/integration/validators.spec.js

```diff
@@ -40,8 +51,21 @@ describe('Validators routes', () => {
         .expect(200)
         .expect('Content-Type', 'application/json; charset=utf-8')
 
+      const [latestBlock] = blocks.filter((block) => block.validator === validator.pro_tx_hash).toReversed()
+
       const expectedValidator = {
-        proTxHash: validator.pro_tx_hash
+        proTxHash: validator.pro_tx_hash,
+        blocksCount: blocks.filter((block) => block.validator === validator.pro_tx_hash).length,
+        lastProposedBlockHeader: latestBlock
+          ? {
```

Comment:
latestBlock should be either null or object, then we don't need the ternary

Reply:
✅

Reply:
Maybe like this?

```
const lastProposedBlockHeader =  blocks.filter(....)[0] ?? null

const expectedValidator = {
  proTxHash: validator.pro_tx_hash,
  proposedBlocksAmount: blocks.filter((block) => block.validator === validator.pro_tx_hash).length,
  lastProposedBlockHeader,
  hash,
  height
....
```

Reply:
Or even better, put inside the exepectValidator as well. just like proposedBlocksAmount

Reply:
✅